--- firefox-128.3.1/js/src/jit/MacroAssembler.h	2024-10-12 14:53:26.916157608 +0200
+++ firefox-128.3.1-jit/js/src/jit/MacroAssembler.h	2024-10-12 14:55:15.445356475 +0200
@@ -3905,16 +3905,16 @@
 
   void wasmCheckSlowCallsite(Register ra, Label* notSlow, Register temp1,
                              Register temp2)
-      DEFINED_ON(x86, x64, arm, arm64, loong64, mips64, riscv64);
+      DEFINED_ON(x86, x64, arm, arm64, loong64, mips64, riscv64, ppc64);
 
   // Places slow class marker for tail calls.
   void wasmMarkCallAsSlow()
-      DEFINED_ON(x86, x64, arm, arm64, loong64, mips64, riscv64);
+      DEFINED_ON(x86, x64, arm, arm64, loong64, mips64, riscv64, ppc64);
 
   // Combines slow class marker with actual assembler call.
   CodeOffset wasmMarkedSlowCall(const wasm::CallSiteDesc& desc,
                                 const Register reg)
-      DEFINED_ON(x86_shared, arm, arm64, loong64, mips64, riscv64);
+      DEFINED_ON(x86_shared, arm, arm64, loong64, mips64, riscv64, ppc64);
 #endif
 
   // WasmTableCallIndexReg must contain the index of the indirect call.  This is
